### ISSUE TITLE
Alexa-Spotify Adapter Play-Pause Issue

### ISSUE THREAD

**Issue by Alexa-AUH** [alexa/avs-device-sdk#1806]
**IMPORTANT**: Before you create an issue, please take a look at our [Issue Reporting Guide](https://github.com/alexa/avs-device-sdk/wiki/Issue-Reporting-Guide).

## Briefly summarize your issue:
Spotify connect Pause not is in sync with Speaker:– App shows paused status on mobile but in Speaker sound is playing for more than 3-4 secs.

## What is the expected behavior?
There should be syncing between Spotify Connect Application and Speaker.

## What behavior are you observing?
Whenever the user pauses a song using the Spotify Connect application, Song should be stopped without any delay.

## Provide the steps to reproduce the issue, if applicable:
Play a song from Spotify and pause the song.

## Tell us about your environment:
Linux
avs-device-sdk-1.18 
Spotify adapter v1.18 (Spotify-1.18.tar.gz) 
Spotify eSDK 3.122. 

### What version of the AVS Device SDK are you using?
avs-device-sdk-1.18

### Tell us what hardware you're using:
Linux-Ubuntu 16.04

### Tell us about your OS (Type & version):
Linux

[Spotify_Pause_Issue.log](https://github.com/alexa/avs-device-sdk/files/5364272/Spotify_Pause_Issue.log)

**Comment by Alexa-AUH** [comment]
To fix this issue, I tried to alter the avs-device-sdk(1.18 version)/CapabilityAgents/ExternalMediaPlayer/src/ExternalMediaPlayerAdapters/SpotifyESDKAdapter.cpp file and the change is working fine till now.

Do let me know if this change is applicable or not.

I attached both the original  (ORG_SpotifyESDKAdapter.cpp) as well as the updated (Changed_SpotifyESDKAdapter.cpp) file.
[File_Change.zip](https://github.com/alexa/avs-device-sdk/files/5416408/File_Change.zip)
